code/src/WijmoProvider/Columns/DateColumn.ts

```diff
@@ -54,5 +53,13 @@ namespace WijmoProvider.Column {
 
             super.build();
         }
+
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+        public changeProperty(propertyName: string, propertyValue: any): void {
+            switch (propertyName) {
+                default:
+                    super.changeProperty(propertyName, propertyValue);
+            }
+        }
```

Comment:
This is unnecessary. You can remove completely the change property from here :)

Reply:
Removed it :)